yarn-project/pxe/src/note_processor/note_processor.test.ts

```diff
@@ -58,7 +58,9 @@ class MockNoteRequest {
   encrypt(): EncryptedL2NoteLog {
     const ephSk = GrumpkinScalar.random();
     const recipient = AztecAddress.random();
-    const log = this.logPayload.encrypt(ephSk, recipient, this.ivpk, this.ovKeys);
+    const logWithoutNumPublicValues = this.logPayload.encrypt(ephSk, recipient, this.ivpk, this.ovKeys);
+    // We prefix the log with an empty byte indicating there are 0 public values.
+    const log = Buffer.concat([Buffer.alloc(1), logWithoutNumPublicValues]);
```

Comment:
Ideally we would have a type which would encapsulate `[num pub values, encrypted log, public values]` but everything is impermanent and all this code is going to die soon so I decided to not spend time on this.

Reply:
The horrors persist

Reply:
I apologize